contributing_guidelines.md

```diff
@@ -1,414 +1,89 @@
+- Document your reasoning behind the changes. Explain why you wrote the code in the way you did. The code should explain what it does.
+- If there's an existing issue, reference to it by adding something like `References/Closes/Fixes/Resolves #123`, where 123 is the issue number. 
+- Please fill out the PR Template when making a PR.
 
-The CircuitVerse Interactive Book's sources are hosted in a GitHub repository. Besides the web interface provided by GitHub, other tools to manipulate git repositories can be used:
+> Please note: maintainers may close your PR if it has gone stale or if we don't plan to merge the code.
 
--   [Git](https://git-scm.com/)
+## Pull request reviews
 
+- Requested Changes must be resolved (with code or discussion) before merging.
+- If you make changes to a PR, be sure to re-request a review.
+- Don't repeadetely tag someone(may be it is not the right time to review your PR), be patient.
+- Do not 'resolve conversation' unnecessary raised by community member or any workflows tools(codeclimate or hound) as they may have some purpose, try to resolve the request changes and if any help wanted tag communtiy member to give views about that.
 
-### Jekyll
+#### PR Lebels
 
-The Interactive Book is rendered using the Jekyll Static Site Generator (SSG). The GitHub repository is able to create a live version of the book using the continuous integration (CI) / continuous deployment (CD) workflows.
+- [under-review](https://github.com/CircuitVerse/Interactive-Book/labels/under-review) labeled PR are under review by core team.
+- [waiting for contributor](https://github.com/CircuitVerse/Interactive-Book/labels/waiting-for-contributor) labeled PR are meant to waiting for contributor to respond.
+- [waiting for design](https://github.com/CircuitVerse/Interactive-Book/labels/waiting-for-design) labeled PR are meant to waiting for review from UI/UX core team.
+- [no activity](https://github.com/CircuitVerse/Interactive-Book/labels/no%20activity) labeled PR are meant to have no activity in the PR from since a while.
+- [blocked](https://github.com/CircuitVerse/Interactive-Book/labels/blocked) labeled PR are meant to not to go ahead for review.
+- [do not merge](https://github.com/CircuitVerse/Interactive-Book/labels/do%20not%20merge) labeled PR are meant to not to merge the PR right now(may be later).
+- [awaiting-approval](https://github.com/CircuitVerse/CircuitVerse/labels/awaiting-approval) labeled PR are meant to be waiting for other communtiy members.
 
-To work locally, you will need to setup a Jekyll development environment. It is posible to setup Jekyll natively for your OS or run it in a docker container.
+## Community
+Discussions about Interactive Book issues and features take place on the repository's [Discussions](https://github.com/CircuitVerse/CircuitVerse/discussions) sections. Anybody is welcome to join these conversations. See the [README](README.md) for more information on communication channels.
```

Comment:
I think slack should be here instead of discussion

Reply:
Done